src/app/src/pages/redteam/setup/components/Targets/CustomTargetConfiguration.tsx

```diff
@@ -23,16 +23,23 @@ const CustomTargetConfiguration = ({
   setRawConfigJson,
   bodyError,
 }: CustomTargetConfigurationProps) => {
-  const [targetId, setTargetId] = useState(selectedTarget.id || '');
+  const [targetId, setTargetId] = useState(selectedTarget.id?.replace('file://', '') || '');
 
   useEffect(() => {
-    setTargetId(selectedTarget.id || '');
+    setTargetId(selectedTarget.id?.replace('file://', '') || '');
   }, [selectedTarget.id]);
 
   const handleTargetIdChange = (e: React.ChangeEvent<HTMLInputElement>) => {
-    const newId = e.target.value;
-    setTargetId(newId);
-    updateCustomTarget('id', newId);
+    const value = e.target.value;
+    setTargetId(value);
+
+    let idToSave = value;
+    if (value && (value.endsWith('.py') || value.endsWith('.js'))) {
```

Comment:
should also handle the case where they use a non standard function name, eg. myprovider.py:func See --detectOpenHandles

Reply:
good call